docs/kpm_oci-zh.md

```diff
@@ -2,75 +2,110 @@
 
 从 kpm v0.2.0 版本开始，kpm 支持通过 OCI Registries 保存和分享 KCL 包。
 
+## kpm registry
+
+kpm 支持使用 OCI Registry 保存和分享 KCL 包，kpm 默认使用 ghcr.io 保存 KCL 包。
+
+kpm 默认 registry - https://github.com/orgs/KusionStack/packages
+
+可以在 kpm 的配置文件中调整 Registry 的地址和仓库名。kpm 的配置文件位于 `$KCL_PKG_PATH/.kpm/config.json`, 如果没有设置环境变量 `KCL_PKG_PATH` 的值，则默认保存在 `$HOME/.kcl/kpm/.kpm/config.json`。
+
+配置文件的默认内容如下：
+
+```json
+{
+    "DefaultOciRegistry":"ghcr.io",
+    "DefaultOciRepo":"kusionstack"
+}
+```
+
 ## 快速开始在 kpm 中使用 OCI Registry
 
+在接下来的内容中，我们将使用 `localhost:5001` 作为示例 OCI Registry，并且为这个 OCI Registry 添加了一个账户 `test`，密码是 `1234`。
+
 ### kpm login
 
-使用 kpm 登陆 OCI registry，需要手动输入密码。
+你可以通过以下四种方式使用 `kpm login`。
+
+#### 1. 使用账户和密码登陆 OCI Registry
+
+```shell
+$ kpm login -u <account_name> -p <password> <oci_registry>
+Login succeeded
+```
+
+<img src="./gifs/kpm_login.gif" width="600" align="center" />
+
+#### 2. 使用账户登陆 OCI Registry，并且交互式输入密码
 
 ```shell
 $ kpm login -u <account_name> <oci_registry>
-Password: # 手动输入密码。
+Password:
+Login succeeded
+```
+
+<img src="./gifs/kpm_login_with_pwd.gif" width="600" align="center" />
+
+#### 3. 交互式输入账户和密码登陆 OCI Registry
+
+```shell
+$ kpm login <oci_registry>
+Username: <account_name>
+Password:
 Login succeeded
 ```
 
-- `<account_name>` 是用来登陆 OCI registry 的账户名称。
-- `<oci_registry>` 是 OCI registry 的 URL，例如：'docker.io', 'ghcr.io'等。
+<img src="./gifs/kpm_login_with_both.gif" width="600" align="center" />
 
 ### kpm logout
 
-使用 kpm 登出 OCI registry
+你可以使用 `kpm logout` 退出一个 OCI Registry。
 
 ```shell
-kpm logout <registry>
+$ kpm logout <registry>
```

Comment:
For one-line shell scripts, do not need to add `$`

Reply:
fixed.